tests/test.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+BINARY=build/bin/minishift
+ISO=file:///root/payload/build/minishift-centos7.iso
+EXTRA_FLAGS="--show-libmachine-logs"
```

Comment:
This will make running tests locally very difficult. I think we still want to keep it easier for contributors to run the test locally. WDYT?